src/services/repairRequest.ts

```diff
@@ -1,26 +1,45 @@
+import { RepairRequestImage } from "@prisma/client";
+
+import { presignImage, presignImages } from "@/lib/presign-image";
 import userService from "@/services/user";
 import { RepairRequest, RepairRequestResponse } from "@/types";
 
+/**
+ * Converts a list of RepairRequest to RepairRequestResponse
+ * @param repairRequests
+ * @param withImage If you need the image presigned urls
+ * @returns
+ */
 const toClientResponse = async (
-  repairRequests: RepairRequest[]
+  repairRequests: Array<RepairRequest & { images?: RepairRequestImage[] }>, // TODO: fix type later
+  withImage = false
```

Comment:
Can we check for images by checking `repairRequests.images` instead of this?

Reply:
Consider scenario like listing repair-request with thumbnail but not all the image, we can't do that.